app/services/fraud_review_check.rb

```diff
@@ -0,0 +1,29 @@
+class FraudReviewCheck
+  class NullFraudReviewUser
+    def fraud_review_pending_profile; end
+
+    def fraud_rejection_profile; end
+  end
```

Comment:
Good to see this pattern in use. @jmax-gsa, look!

Reply:
IMO this is not a super clear use case for this pattern? The `fraud_review_eligible?` method already has the long nil-safe `&.` chain that the null object pattern would ostensibly avoid? So I feel like it would be easier to remove and just make sure all the method calls have `&.`?

Reply:
The idea is to eventually move those methods out of the user. I introduced the null object in a branch where I had removed them from the user but I need a follow-up to make some changes to decisioning to fully yank them. I can take them off of here until that is done if you think this makes a mess

Reply:
I forgot about `NullUser`. Let me revisit and see what I can do.